schemas/org.w3/PerformanceNavigationTiming/jsonschema/1-0-0

```diff
@@ -0,0 +1,231 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for page navigation performance, based on the PerformanceNavigationTiming interface (see https://w3c.github.io/navigation-timing/)",
+    "self": {
+        "vendor": "org.w3",
+        "name": "PerformanceNavigationTiming",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "decodedBodySize": {
+            "type": [ "integer", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A number that is the size (in octets) received from the fetch (HTTP or cache) of the message body, after removing any applied content encoding."
+        },
+        "encodedBodySize": {
+            "type": [ "integer", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A number representing the size (in octets) received from the fetch (HTTP or cache), of the payload body, before removing any applied content encodings."
+        },
+        "redirectStart": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp that represents the start time of the fetch which initiates the redirect in milliseconds."
+        },
+        "redirectEnd": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
+            "description": "A DOMHighResTimeStamp immediately after receiving the last byte of the response of the last redirect in milliseconds."
+        },
+        "fetchStart": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": 0,
```

Comment:
The main resource for the issue I found is on the mozzila [bug reporting](https://bugzilla.mozilla.org/show_bug.cgi?id=1429422). 

Replicated that by opening Firefox from my console.
![image](https://user-images.githubusercontent.com/15251081/234598597-72f3a43b-3ef0-4ce4-bc17-b8f764140100.png)

This would of course create validation errors. Which in my view we do not expect in production.

@istreeter Does it have have any drawback in setting this values on minimum of negative the maximum ?

Reply:
From loader's point of view, it is fine to use a negative number for the minimum.  Anything down to -2147483648.

Reply:
Thanks Ian